Directory.Build.props

```diff
@@ -0,0 +1,13 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <Authors>Liam Riddell,Sebastian Stehle</Authors>
+    <Description>An unofficial port of MJML (by MailJet) to .NET Standard. This project is currently in experimental state and should not be used in a production environment.</Description>
+    <Copyright>MIT</Copyright>
+    <PackageTags>email,mjml</PackageTags>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <PackageProjectUrl>https://github.com/LiamRiddell/MJML.NET</PackageProjectUrl>
```

Comment:
We'll need to change this to point to the right location.

Reply:
What is the right location?

Reply:
Good question. I don't mind :)

Reply:
Okay, then we can just keep it here. But can you add your nuget key to the secrets?

Reply:
@SebastianStehle, I can't access repository settings to add the secret.

Reply:
I tried to add you again, but it seems I cannot give you permissions. Can you send it to sebastian@squidex.io?

Reply:
Is my user marked as admin?

Reply:
I don't know how.